backend/cinema_service/settings.py

```diff
@@ -30,9 +30,9 @@
 
 ALLOWED_HOSTS = []
 
-INTERNAL_IPS = [
-    "127.0.0.1",
-]
+# INTERNAL_IPS = [
```

Comment:
why do you need this comment?

Reply:
I had so many troubles with this task and tried various things to solve it. And I just forgot to change it back.